tests/components/technove/__init__.py

```diff
@@ -1 +1,17 @@
 """Tests for the TechnoVE integration."""
+from unittest.mock import patch
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def setup_with_selected_platforms(
```

Comment:
would be better to set a fixture in `conftest` to load the integration once.
Not seeing why we need to load single platforms.

Reply:
For the snapshot test, you still want to have the tests separated per platform, but not have every platform snapshot every entity

Reply:
Sure, was only referring to setting up the integration.

Reply:
I know, but when I added Withings sensor snapshot tests I checked if they were the right platform when snapshotting, but I was told I should patch PLATFORMS instead

Reply:
I guess anyone's preference...